src/selic-entries.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { selicEntries } from "./selic-entries";
+
+describe("selicEntries", () => {
+    const desiredSelicEntrie = [
+        {
+            "data": '08/03/2024',
+            "valor": '0.041957',
+        }
+    ];
```

Comment:
This shouldn't be here. When declared here, it'll become global to all the tests in this scope.